combat-trainer.lic

```diff
@@ -481,8 +485,6 @@ class SpellProcess
     @osrel_no_harness = settings.osrel_no_harness
     echo("  @osrel_no_harness: #{settings.osrel_no_harness}") if $debug_mode_ct
 
-    @cast_only_to_train = settings.cast_only_to_train
```

Comment:
I'm a little confused about why these changes are still showing up. I've fetched and merged with the remote 'upstream' which is your master, and then created this branch